### DISCUSSION TITLE
CVE-2021-44228 (Apache log4j) issue Atom IO

### DISCUSSION THREAD

**Discussion by dshifflett** [atom/atom#23378] [Q&A]
I have been ask by my security office to confirm if you software is affected by CVE-2021-44228 (Apache log4j) vulnerability. Can you please confirm if this is an issue with Atom IO

**Comment by Franetse7** [comment]
Non mais ajout le Jeton ?

**Reply by KOHLECENTER** [reply]
Yes

**Comment by icecream17** [comment]
I have no idea but the search "org:atom log4j" https://github.com/search?q=org%3Aatom+log4j doesn't return anything important

**Comment by alkikostakou** [comment]
I have the same question from my security team. Does anyone know if the log4J issue is applicable to Atom?
